reference-implementation/lib/abstract-ops/readable-streams.js

```diff
@@ -257,13 +258,18 @@ function ReadableStreamPipeTo(source, dest, preventClose, preventAbort, preventC
     setPromiseIsHandledToTrue(pipeLoop());
 
     function waitForWritesToFinish() {
-      // Another write may have started while we were waiting on this currentWrite, so we have to be sure to wait
```

Comment:
Why this change? The previous version seems a bit more understandable, although both are fairly convoluted...

Reply:
Originally, I was fiddling with `pipeTo()` so that it waits for all of its `read()` and `write()` promises to settle. There was a `waitForReadsAndWritesToFinish()` helper that checked and waited for both, and this pattern made that easier. But that's no longer in the current version, so it's no longer needed.

I'll revert it.